src/MicrosoftTeams.ts

```diff
@@ -588,6 +588,31 @@ namespace microsoftTeams {
     };
   }
 
+  /**
+   * enable print capability
+   */
+  export function enablePrintCapability(): void {
+    ensureInitialized();
+    /**
```

Comment:
NIT: Generally the `/** ... */` multi-line comments only appear at the top of functions. In here you can just use a single line comment `// ...`